themes/tiddlywiki/vanilla/base.tid

```diff
@@ -92,7 +91,7 @@ pre {
 	white-space: pre-wrap;
 	background-color: <<colour pre-background>>;
 	border: 1px solid <<colour pre-border>>;
-	padding: 0 3px 2px;
+	padding: 0.5em;
```

Comment:
@nameanyone, sorry for my partly confusing reply. I mixed up this PR with another. Of course, changing **vanilla/base** would be right place.